packages/amazonq/src/extensionShared.ts

```diff
@@ -61,6 +62,8 @@ export async function activateShared(context: vscode.ExtensionContext) {
     await activateCWChat(context)
     await activateQGumby(extContext as ExtContext)
 
+    await activateDev(context)
```

Comment:
I think activateDev needs to be owned by Toolkit. won' this create duplicate commands?

Reply:
Good catch. activateDev has been removed from this PR, I'll properly implement it in a different one.